ms18-database/team-c/Context/MaasgroepContext.cs

```diff
@@ -352,38 +352,38 @@ private void CreateReceiptApproval(ModelBuilder modelBuilder)
 
 		private void CreatePhoto(ModelBuilder modelBuilder)
 		{
-			modelBuilder.Entity<Photo>().ToTable("photo", "receipt");
-			modelBuilder.Entity<Photo>().HasKey(p => new { p.Id });
+			modelBuilder.Entity<ReceiptPhoto>().ToTable("photo", "receipt");
```

Comment:
Hier zeg je alsnog dat de tabel photo moet gaan heten

Reply:
Als je in een ander schema een tabel photo noemt is dat ok.
Dus een "public"."photo" kan er naast staan (public is het default schema).